common/vision/lasr_vision_pointing/scripts/detect_pointing.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python3
+
+import os
+import cv2
+import rospkg
+import rospy
+import mediapipe as mp
+from cv_bridge import CvBridge
+from lasr_vision_msgs.srv import YoloDetectionRequest, YoloDetection, PointingDirection, PointingDirectionResponse, PointingDirectionRequest
+
+'''using this rather than openpose because it is faster and more accurate'''
+class PointingDetecor:
+    def __init__(self):
+        self.detect_service = rospy.ServiceProxy('/yolov8/detect', YoloDetection)
+        self.service = rospy.Service('/pointing_detection_service', PointingDirection, self.excute)
+        self.counter = 0
```

Comment:
should be removed if the images directory is removed (counter is strictly for storing images)